lib/web_ui/dev/browser_roller.dart

```diff
@@ -118,6 +118,10 @@ class _BrowserRoller {
     for (final _Platform platform in _platforms) {
       await _rollChromium(platform);
       await _rollChromeDriver(platform);
+      // For now, we only test Firefox on Linux.
+      if (platform.os == 'linux') {
+        await _rollFirefox(platform);
+      }
```

Comment:
Instead of fishing linux out of the for loop, it's probably simpler to just call it after the loop passing it linux?

Reply:
That would mean either hard-coding that the linux platform is `_platforms[0]` or copying the platform constructor `_Platform('linux', 'amd64', LinuxPlatformBinding())`